src/odm_sharing/tools/share.py

```diff
@@ -148,7 +150,7 @@ def get_excel_writer(in_name, debug: bool, org: str, outdir: str,
                      outfmt: OutFmt) -> Optional[pd.ExcelWriter]:
     if not debug and outfmt == OutFmt.EXCEL:
         filename = gen_filename(in_name, org, '', 'xlsx')
-        print('writing ' + filename)
+        logging.info('writing ' + filename)
```

Comment:
Doesn't this still log to the console?

Reply:
the python logging doesn't write to stdout like print does, while still allowing us to have a log of what happened (to a file, etc. I may have forgotten to add a log file handler tho). This is good because we don't want to pollute stdout when making a CLI program. Our errors are written to stderr instead, so that's fine. But yeah, I should check/add that log file.

Reply:
an alternative would be to add a `-q`/`--quiet` switch to turn off messages to stdout.